src/config.js

```diff
@@ -13,7 +13,7 @@ function applyConfigOptions (configOps) {
   }
 }
 
-module.exports.load = async (configOps) => {
+module.exports.load = (configOps) => {
```

Comment:
This was hiding errors before because `async` was forcing the execution to be async instead of sync.